README.md

```diff
@@ -1,6 +1,6 @@
- desktop-version
+ Accessibility
 <a name="readme-top"></a>
-Desktop version day2
+Accessibilty day 2
 <div align="center">
   <h3><b>Microverse README Template</b></h3>
 </div>
```

Comment:
- [ ] Your READme file links should be clickable. See the image below showing how it is currently, the table of content links are not clickable.

![image](https://user-images.githubusercontent.com/84156714/216094012-7a54cb90-f9b0-4ce8-a560-5cdba7f76d08.png)